repo2docker/buildpacks/r.py

```diff
@@ -186,28 +185,41 @@ def get_packages(self):
 
         return super().get_packages().union(packages)
 
-    def _get_latest_working_mran_date(self, startdate, max_prior):
+    def get_cran_mirror_url(self, snapshot_date):
+        # Call the API to find out if we have a snapshot available for the given date.
+        # If so, use the URL for that snapshot. If not, fall back to MRAN.
+        snapshots = requests.post(
+            "https://packagemanager.rstudio.com/__api__/url",
```

Comment:
How did you find out about this API? It's not in https://packagemanager.rstudio.com/__api__/swagger/index.html

Reply:
@manics hmm, I found it by looking at network tab in https://packagemanager.rstudio.com/client/#/repos/1/overview